E3SM-3-0/piControl_r1i1p1f1.json

```diff
@@ -0,0 +1,103 @@
+
+  "parent_activity_id": "CMIP",
+
+  "parent_source_id": "E3SM-3-0",
+
+  "parent_variant_label": "r1i1p1f1",
+
+  "parent_time_units": "days since 0001-01-01",
+
+  "branch_method": "standard",
+
+  "branch_time_in_child": 0.0,
+
+  "#note_branch_time_in_parent":"i.e. first historical ensemble branch from 0101-01-01, gives branch time 36500 days. This value is a multiple of 365" ,
+  
+  "branch_time_in_parent": 0.0,
+
+  "#note_institution_id": "institution_id must be registered at https://github.com/WCRP-CMIP/CMIP6_CVs/issues/new ",
+
+  "institution_id": "E3SM-Project",
+
+  "#note_source_id": "source_id (model name) must be registered at https://github.com/WCRP-CMIP/CMIP6_CVs/issues/new ",
+
+  "source_id": "E3SM-3-0",
+
+  "calendar": "noleap",
+
+  "grid": "data regridded to a CMIP6 standard 1x1 degree lonxlat grid from the native grid using an area-average preserving method.",
+
+  "grid_label": "gr",
+
+  "nominal_resolution": "100 km",
+
+  "#output": "Root directory for output (can be either a relative or full path)",
+
+  "outpath": "CMIP6",
+
+  "#note_optional": " **** The following descriptors are optional and may be set to an empty string ",
+
+  "project PI": "Dr. Peter Caldwell (caldwell19@llnl.gov), Dr. Dave Bader (PI until Feb 2025)",
+
+  "data contact": "E3SM-DATA-SUPPORT@LLNL.GOV",
+
+  "license": "CMIP6 model data produced by E3SM-Project is licensed under a Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse for terms of use governing CMIP6 output, including citation requirements and proper acknowledgment. Further information about this data, including some limitations, can be found via the further_info_url (recorded as a global attribute in this file) and at https://pcmdi.llnl.gov/. The data producers and data providers make no warranty, either express or implied, including, but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law.",
+
+  "#note_history": "include the original case name",
+
+  "history": "v3.LR.piControl",
+
+  "comment": "",
+
+  "references": "Xie, S., et al. The Energy Exascale Earth System Model Version 3. Part I: Overview of the Atmospheric Component. ESS Open Archive. April 13, 2025. in review with JAMES; Golaz, J.-C. and co-authors, 2022: The Energy Exascale Earth System Model version 3. Part II: Overview of the coupled system. JAMES, in review concurrent to data publication",
+
+  "#note_CMIP6": " **** The following are set correctly for CMIP6 and should not normally need editing",
+
+  "_controlled_vocabulary_file": "CMIP6_CV.json",
+
+  "_AXIS_ENTRY_FILE": "CMIP6_coordinate.json",
+
+  "_FORMULA_VAR_FILE": "CMIP6_formula_terms.json",
+
+  "_cmip6_option": "CMIP6",
+
+  "mip_era": "CMIP6",
```

Comment:
@TonyB9000 I started to add metadata definition file for v3. could you help to review? Have we decided on what to use for "mip_era"?